src/lib/modules/blockchain_listener/index.js

```diff
@@ -40,6 +43,50 @@ class BlockchainListener {
       this.processLogsApi.logHandler.handleLog({logLevel, message});
     });
   }
+
+  _registerConsoleCommands() {
+    this.embark.registerConsoleCommand({
+      description: 'Enables regular transactions to prevent transactions from getting stuck when using Geth and Metamask',
+      matches: ['regularTxs on'],
+      usage: "regularTxs on",
+      process: (cmd, callback) => {
+        this.events.request('regularTxs:start', callback);
+      }
+    });
+    this.embark.registerConsoleCommand({
+      description: 'Disables regular transactions used to prevent transactions from getting stuck when using Geth and Metamask',
```

Comment:
Another "hacky" way to add the description to the `help` would be to just add one description (for example, on the one above), and then you can have `usage: 'regularTxs on/off'` and the the description can be "Toggles regular transactions to prevent transactions from getting stuck when using Geth and Metamask"

TLDR: the Help command only adds a line when there is a `description`

Reply:
I like it! Updated... and I wouldn't necessarily say it's "hacky" - it's better this way :)

Reply:
Nice. You did it even better than I imagined!